app/i18n/cz/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Dokončit instalaci',
 		'fix_errors_before' => 'Chyby prosím před přechodem na další krok opravte.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
Don't worry, I will provide you some help when this pull request will me merged :). You will be able to edit the file directly on GitHub at https://github.com/FreshRSS/FreshRSS/edit/dev/app/i18n/cz/install.php (but wait for the merge first ;))